orchagent/flexcounterorch.h

```diff
@@ -15,10 +15,14 @@ class FlexCounterOrch: public Orch
     void doTask(Consumer &consumer);
     FlexCounterOrch(swss::DBConnector *db, std::vector<std::string> &tableNames);
     virtual ~FlexCounterOrch(void);
+    bool getPortCountersState();
```

Comment:
>; [](http://example.com/codeflow?start=31&length=1)

Add `const` if it is a `get` function.

Reply:
We can't declare this return value as const for this function since the bool we are returning is changing on run time.

Reply:
```suggestion
    bool getPortCountersState() const;
```
Not the return value, but the whole function

Reply:
Got you, fixed